queries.yaml

```diff
@@ -1859,18 +1886,18 @@ post_process:
       target_attribute: landuse_kind
       cutting_attrs: { sort_key: 'sort_rank', reverse: True }
 
-  # drop landuse_kind=residential or industrial at mid-low zooms, because they
+  # conditionally drop landuse_kind at mid-low zooms, because they
   # interfere a lot with merging, and tend to be not strongly styled unlike
   # military, hospital or park/forest land uses.
   - fn: vectordatasource.transform.drop_properties
     params:
       source_layer: roads
       start_zoom: 8
-      end_zoom: 14
+      end_zoom: 16
       properties:
         - landuse_kind
       where: >-
-        landuse_kind in ('residential', 'industrial')
+        landuse_kind not in ['aerodrome', 'airfield', 'cemetery', 'college', 'conservation', 'forest', 'garden', 'golf_course', 'hospital', 'military', 'national_park', 'nature_reserve', 'park', 'parking', 'protected_area', 'runway', 'school', 'taxiway', 'university']
```

Comment:
This can be verified in 512 px tile **14-9612-6755** looking at شارع المطار street. The yellow segment on left has the same properties, except the `landuse_kind` value is null instead of residential.

We could have extended the end_zoom to accomplish the same thing for this tile... but the logic above also accomplishes the same thing while also removing `landuse_kind` = grass and other irrelevant details.

<img width="1218" alt="image" src="https://user-images.githubusercontent.com/853051/153134924-97d7d2c6-983e-4dbc-9d07-e3d632e4c516.png">

Reply:
Fixed in latest build:

<img width="1156" alt="image" src="https://user-images.githubusercontent.com/853051/153689653-69088be1-cda6-41ff-9666-2719b4432640.png">